concepts/05 Widgets/DataGrid/45 Grouping/10 User Interaction/10 Group Data.md

```diff
@@ -47,6 +47,71 @@ Assigning **true** to the **grouping**.[contextMenuEnabled](/api-reference/10%20
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <Dx{WidgetName} ...
```

Comment:
`{WidgetName}` and `{widget-name}` should be replaced with, correspondingly, PascalCase and kebab-case versions of the widget name. In our case, these are `DataGrid` and `data-grid`